src/main/java/org/cobspec/CobSpec.java

```diff
@@ -53,7 +53,7 @@ public static void main(String[] args) throws IOException {
 
         app.get("/method_options", (request) -> Response.create());
         app.post("/method_options", (request) -> Response.create());
-        app.put("/method_options", fileSystemController::write);
+        app.put("/method_options", (request) -> Response.create());
```

Comment:
It's just to conform to cob_spec.  It makes this request, but if a resource is created (like it should for a PUT request) another test fails.  The method_options routes aren't intended to do anything.  They only exist to show that an OPTIONS request correctly indicates methods that are supported (even if those methods don't behave correctly).

There are several instances of cob_spec not following the HTTP spec perfectly.  Both of the PRs I submitted on cob_spec address similar issues.